packages/web-app-files/src/components/PublicLinksSidebar/EditPublicLink.vue

```diff
@@ -22,57 +21,49 @@
           @roleSelected="$_selectRole"
         />
       </oc-grid>
-      <div class="oc-mb uk-grid-small uk-flex uk-flex-middle" uk-grid>
+      <div class="oc-mb uk-grid-small uk-flex" uk-grid>
         <div v-if="$_expirationDate" class="uk-width-1-1 uk-width-2-5@m">
-          <label class="oc-label" for="oc-files-file-link-expire-date">
-            <span v-translate>Expiration date:</span>
-            <translate v-if="$_expirationDate.enforced" tag="em">(required)</translate>
-          </label>
           <div class="uk-position-relative">
             <oc-datepicker
               id="oc-files-file-link-expire-date"
               :key="'oc-datepicker-' + expireDate"
-              :class="{ 'uk-form-danger': !$_expirationIsValid }"
+              :label="
+                `${$gettext('Expiration date')} ${
+                  !!$_expirationDate.enforced ? `(${$gettext('required')})` : ''
+                }`
+              "
               :date="expireDate"
               :max-datetime="$_maxExpirationDate"
               :min-datetime="$_minExpirationDate"
-              :placeholder="placeholder.expireDate"
               @input="expireDate = $event"
             />
-            <div
+            <a
```

Comment:
Would it make sense to use an `oc-button` instead of an `a` tag here? You can use it with `appearance="raw"`.